app/components/machine/driver-custom/component.js

```diff
@@ -79,11 +110,21 @@ export default Ember.Component.extend(ManageLabels, {
 
   registrationCommandWindows: function() {
     let url = this.get('model.registrationUrl');
-
-    return `New-Item -Path 'C:\\Program Files\\rancher' -Type Directory
-Invoke-WebRequest -UseBasicParsing 'https://github.com/rancher/agent/releases/download/v0.6.0/agent.exe' -OutFile 'C:\\Program Files\\rancher\\agent.exe'
-& 'C:\\Program Files\\rancher\\agent.exe' -register-service ${url}
-Start-Service rancher-agent`;
-  }.property('model.command'),
+    let cattleIp = this.get('cattleAgentIp');
+    let env = Util.addQueryParams('', this.get('model.labels')||{});
+    let cmd1=`New-Item -Path 'C:\\Program Files\\rancher' -Type Directory`;
+    let cmd2 = this.get('model.windows-command');
```

Comment:
Please tell me this is a typo and there's not really a field called `windows-command` in the API?  JSON fields are camelCase, hyphen is ambiguous and requires quoting in people's code

Reply:
@orangedeng let's change this to `windowsCommand`